contracts/tools/Swaps.sol

```diff
@@ -106,11 +148,11 @@ contract DVP is Ownable, ERC1820Client, IERC1400TokensRecipient, ERC1820Implemen
    * @param expirationDate Expiration date of the trade.
    * @param tokenAddress1 Address of the first token smart contract.
    * @param tokenValue1 Amount of tokens to send for the first token.
-   * @param tokenId1 ID/partition of the first token.
+   * @param tokenId1 ERC721ID/holdId/partition of the first token.
```

Comment:
Can we find another name for this parameter? Example: tokenIdOrHoldIdOrPartition